public/static/locales/cy/pageTitles.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Should the Welsh version have capital letters at the start of each word, as the English does?
i.e. Manylion Eraill

Reply:
Haven't done this since we couldn't be sure where the capitals should be on some of the translations

Reply:
I guess the translators can tell us if they think they should be added

Reply:
Done now with the help of Google Translate to determine capitalisation.  Can be checked by FSA in Staging.